### PR TITLE
Fix inconsistent titles

### PR DESCRIPTION
**Pull request by mfuji09**
#### Summary
Fix inconsistent titles `Location: XXX` to `Location.XXX`

#### Motivation
I couldn't find a reason to title as so.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/location/ancestororigins/index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: ancestorOrigins'
+title: Location.ancestorOrigins
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/ancestororigins/index.md:2]:
```suggestion
title: location.ancestorOrigins
```

File: files/en-us/web/api/location/assign/index.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 'Location: assign()'
+title: Location.assign()
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/assign/index.md:2]:
```suggestion
title: location.assign()
```

File: files/en-us/web/api/location/hash/index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: hash'
+title: Location.hash
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/hash/index.md:2]:
```suggestion
title: location.hash
```

File: files/en-us/web/api/location/host/index.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: host'
+title: Location.host
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/host/index.md:2]:
```suggestion
title: location.host
```

File: files/en-us/web/api/location/hostname/index.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: hostname'
+title: Location.hostname
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/hostname/index.md:2]:
```suggestion
title: location.hostname
```

File: files/en-us/web/api/location/href/index.md [hunk 6]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: href'
+title: Location.href
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/href/index.md:2]:
```suggestion
title: location.href
```

File: files/en-us/web/api/location/origin/index.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: origin'
+title: Location.origin
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/origin/index.md:2]:
```suggestion
title: location.origin
```

File: files/en-us/web/api/location/password/index.md [hunk 8]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: password'
+title: Location.password
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/password/index.md:2]:
```suggestion
title: location.password
```

File: files/en-us/web/api/location/pathname/index.md [hunk 9]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: pathname'
+title: Location.pathname
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/pathname/index.md:2]:
```suggestion
title: location.pathname
```

File: files/en-us/web/api/location/port/index.md [hunk 10]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: port'
+title: Location.port
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/port/index.md:2]:
```suggestion
title: location.port
```

File: files/en-us/web/api/location/protocol/index.md [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: protocol'
+title: Location.protocol
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/protocol/index.md:2]:
```suggestion
title: location.protocol
```

File: files/en-us/web/api/location/reload/index.md [hunk 12]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: reload()'
+title: Location.reload()
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/reload/index.md:2]:
```suggestion
title: location.reload()
```

File: files/en-us/web/api/location/replace/index.md [hunk 13]
```diff
@@ -1,9 +1,9 @@
 ---
-title: 'Location: replace()'
+title: Location.replace()
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/replace/index.md:2]:
```suggestion
title: location.replace()
```

File: files/en-us/web/api/location/search/index.md [hunk 14]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: search'
+title: Location.search
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/search/index.md:2]:
```suggestion
title: location.search
```

File: files/en-us/web/api/location/tostring/index.md [hunk 15]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: toString()'
+title: Location.toString()
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/tostring/index.md:2]:
```suggestion
title: location.toString()
```

File: files/en-us/web/api/location/username/index.md [hunk 16]
```diff
@@ -1,5 +1,5 @@
 ---
-title: 'Location: username'
+title: Location.username
```
Comment (sideshowbarker) [inline files/en-us/web/api/location/username/index.md:2]:
```suggestion
title: location.username
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Review by sideshowbarker — APPROVED** [review]

**Comment by mfuji09** [comment]
@sideshowbarker Thank you for correcting!

**Comment by sideshowbarker** [comment]
> @sideshowbarker Thank you for correcting!

Per the norms for the site, it wasn’t completely incorrect the way you’d changed it — though that said, the site norms still have the known problem of allowing (or even preferring) incorrect syntax that misleads developers into thinking something’s a static property when it’s actually not.

But outside of that, in cases where we have something that’s actually a global object (`navigator`, `window`, `location`), it seems more helpful for developers if we use the actual object rather than the interface name.

**Pull request merged**